src/net.h

```diff
@@ -373,8 +373,8 @@ class CConnman
     static bool NodeFullyConnected(const CNode* pnode);
 
     // Network usage totals
-    CCriticalSection cs_totalBytesRecv;
-    CCriticalSection cs_totalBytesSent;
+    Mutex cs_totalBytesRecv;
+    Mutex cs_totalBytesSent;
```

Comment:
In Bitcoin, RecursiveMutex is used.

Reply:
they have only one mutex. other is changed to atomic 
```    
    // Network usage totals
    mutable Mutex m_total_bytes_sent_mutex;
    std::atomic<uint64_t> nTotalBytesRecv{0};
    ```